### PR TITLE
Add divergence()

### PR DESCRIPTION
**Pull request by koheiw**
Add divergence() for #26 .

### CODE REVIEW

**Comment by koheiw** [comment]
@contefranz If you are happy with the `divergence()`, please approve this PR.

**Comment by contefranz** [comment]
> @contefranz If you are happy with the `divergence()`, please approve this PR.

I am happy, but left a comment on it. How about adding a similar method for the seededlda method and not just the lda? If you want to split the work in two, I'll approve the PR. Just let me know.

**Comment by koheiw** [comment]
I think I cannot see your comments until you click "Request change".

File: R/utils.R [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#' @param x a LDA model fitted by [textmodel_seededlda()] or [textmodel_lda()]
+#' @details
+#' `divergence()` computes the average Kullback–Leibler distance between all the
+#' pairs of topic vectors in `x$phi`. The divergence score maximizes when the
+#' chosen number of topic `k` is optimal (Deveaud et al., 2014).
+#' @references Deveaud, Romain et al. (2014). "Accurate and Effective Latent
+#'   Concept Modeling for Ad Hoc Information Retrieval".
+#'   doi:10.3166/DN.17.1.61-84. *Document Numérique*.
+#' @export
+divergence <- function(x) {
```
[the remark this answers is not in the document]
Comment (koheiw) [reply R/utils.R:13]: It works with seededlda too. 

https://github.com/koheiw/seededlda/blob/857f9f9b078f14605b1bc09dcea127ea4d3e0c40/tests/testthat/test-textmodel_seededlda.R#L180-L191

**Review by contefranz — APPROVED** [review]
Sounds good to me then!

**Comment by koheiw** [comment]
Thanks!

**Pull request merged**